8.x/theme-development/helpers.md

```diff
@@ -474,38 +608,50 @@ With the following code, you can see the products in the shopping cart.
 </ul>
 ```
 
-Output
-```
-this customer has 1 product in his basket.
+Output:
 
-شیر كشویی فلنجدار چدنی زبانه لاستیكی میراب 50،"2 ،PN10(31) => 2 :sum (2440000)
+```html
+<p>this customer has 1 product in his basket.</p>
+
+<ul>
+    <li>
+        product name => 2 :sum (2440000)
+    </li>
+</ul>
 ```
-With foreach, the characteristics of each product can be displayed in each line. With `product_id`, the product id is shown, with `count` the number of that product, and by passing true boolean, the product object can also be displayed, and the `count` is multiplied by the `latest price` to calculate the product price.
-______________________________
 
+As the `true` value is passed to the `get_local_cart` function, the product object will be returned too.
+To calculate the product price, the `count` and the `latest price` are considered.
+
+---
+
+#### get_system_user
 
-### `get_system_user`
 ```php
 function get_system_user(string $guard = null): ?SystemUser
 ```
-The structure of Larammerce user management is done according to the user object.
-There are two users in the system: 
+According to the Larammerce user management structure, there are two types of users in the system: 
 
-1- System User  
+1- System User(admins)
 
 2- Customer user
 
-This function returns system users who are admins in this system.
-______________________________
-
+This function returns the system user object if the logged-in user is a system user(admin), otherwise returns `null`.
+---
```

Comment:
_**WARNING**_  -> Delete these `---` signs.

Reply:
@ShammasAmir  why?

Reply:
@ShammasAmir Yes, i noticed and resolved it. thank you